spec/features/users/login_spec.rb

```diff
@@ -0,0 +1,17 @@
+# require 'rails_helper'
+
+# describe "the signin process", type: :feature do
+#   before :each do
+#     User.create(email: 'me@example.com', name: "ankole", password: "qwerty")
+#   end
+
+#   it "signs me in" do
+#     visit "users/sign_in"
+#     fill_in "Email", with: "me@example.com"
+#     fill_in "Password", with: "qwerty"
+#     fill_in "Username", with: "ankole"
+#     click_button "Log in"
+#     expect(current_path).to eq(root_path)
+#     expect(page).to have_text('Signed in successfully.')
+#   end
+# end
```

Comment:
- [ ] As requested by the previous reviewer, please remove all unnecessary titles since they are empty  or add details to them (Prerequisites, run tests Deployment ....) 🚮